nl_server/routes.py

```diff
@@ -92,3 +93,9 @@ def search_verbs():
 @bp.route('/api/embeddings_version_map/', methods=['GET'])
 def embeddings_version_map():
   return json.dumps(current_app.config[config.NL_EMBEDDINGS_VERSION_KEY])
+
+
+@bp.route('/api/load/', methods=['GET'])
+def load():
```

Comment:
also nice to have another param for the custom embedding data path.

Reply:
I added a TODO in the other file based on what we decide about always readying "version.txt".  Based on that, will add in follow on PR.